eco.Rmd

```diff
@@ -17,7 +20,9 @@ structure:
 7. discussion: what could be done better or alternatives and again emphasizing that the methods shown
```

Comment:
Sounds good. Again, my only concern is that it's too many sections but better to have too many ideas and too much content than too few and too little at this stage - look forward to reading these sections!